### PR TITLE
Converted the models repo to TF 1.0 using the upgrade script

### PR DESCRIPTION
**Pull request by nealwu**

### CODE REVIEW

**Comment by nealwu** [comment]
There are likely some issues with this PR for models that have already been upgraded or partially upgraded to 1.0 via other PRs, since the upgrade script will incorrectly (re-)reorder the arguments for `tf.concat` and `tf.split`. Will look into the PRs to make sure these are fixed.

@aselle, are there other functions where this type of thing could occur?

**Comment by nealwu** [comment]
OK, had to do a lot of manual editing but this should be good to go now.

**Comment by TheodoreGalanos** [comment]
Hi @nealwu I'm a bit confused, there's been a lot of PRs concerning these models.

Am I right to believe it is safe to clone and build again or should I wait for an upcoming merge?

I'm mostly interested on syntaxnet which had a few issues after the update.

Kind regards,
Theodore.

**Comment by nealwu** [comment]
@TheodoreGalanos this PR won't affect syntaxnet. The reason it makes a change to syntaxnet code is to explicitly specify the `axis` and `values` arguments for `tf.concat`, which prevents the upgrade script from mistakenly re-ordering the arguments in the future.

**Comment by studynon** [comment]
@nealwu In [#1063](https://github.com/tensorflow/models/pull/1063), the upgrade script has solved most general conversions of ‘textsum’ in ‘tensorflow/models’.  
However, there are some specific modifications which should be manually dealt with.
They are committed in [#1119](https://github.com/tensorflow/models/pull/1119) and listed as follow:
seq2seq_attention_model.py 
	----- tf.nn.rnn_cell.LSTMCell-->tf.contrib.rnn.LSTMCell
	----- tf.nn.bidirectional_rnn-->tf.contrib.rnn.static_bidirectional_rnn
	-----tf.nn.seq2seq-->tf.contrib.legacy_seq2seq
	-----in tf.nn.sampled_softmax_loss(), change the order of (inputs,labels)—>(labels, inputs),
seq2seq_lib.py
	-----tf.op_scope(values, name, default_name) is deprecated, use tf.name_scope(name, default_name, values)

Moreover, if the ‘textsum’ is expected to run in Python 3.x, the following changes should be done:
data_convert_example.py
	-------tf_example.features.feature[key].bytes_list.value[0] --> 
                   tf_example.features.feature[key].bytes_list.value[0].decode()
	-------tf_example.features.feature[k].bytes_list.value.extend([v]) --> 
                    tf_example.features.feature[k].bytes_list.value.extend([v.encode()])
batch_reader.py
	-------import Queue --> import queue as Queue
	-------xrange --> range
beam_search.py
	-------xrange --> range
seq2seq_attention_model.py
	------ xrange --> range
seq2seq_attention_decode.py
	------ xrange --> range

**Comment by nealwu** [comment]
@studynon All of the changes in the first half of your post have been submitted in separate pull requests. If you would like to add Python 3 compatibility (while maintaining Python 2 compatibility), feel free to submit a separate pull request for that.
